gitlab/cmd/receive_adapter/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2020 The TriggerMesh Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	webhooks "gopkg.in/go-playground/webhooks.v3"
+	gitlab "gopkg.in/go-playground/webhooks.v3/gitlab"
+	"knative.dev/eventing-contrib/gitlab/pkg/adapter"
+)
+
+const (
+	// Environment variable containing the HTTP port
+	envPort = "PORT"
+
+	// Environment variable containing Gitlab secret token
+	envSecret = "GITLAB_SECRET_TOKEN"
+)
+
+func main() {
+	sink := flag.String("sink", "", "uri to send events to")
+
+	flag.Parse()
+
+	if sink == nil || *sink == "" {
```

Comment:
(nit) The check doesn't hurt, but `sink` will never be `nil` here because the `flag` lib always allocates memory.